File: Beginner/Java/Array/MoveZeros.java
class Solution {
    public void moveZeroes(int[] nums) {
        int pos = 0;
        for (int i = 0; i < nums.length; i ++) {
            if (nums[i] != 0) {
                if (nums[pos] == 0) {
                    nums[pos ++] = nums[i];
                    nums[i] = 0;
                }
                else
                    pos ++;
            }
        }
    }
}

File: Beginner/Python/Array/MoveZeros.py
class Solution(object):
    def moveZeroes(self, nums):
        """
            :type nums: List[int]
            :rtype: void Do not return anything, modify nums in-place instead.
            """
        nums[:] = sorted(nums, lambda x, y: -1 if y == 0 else 0)
